packages/site/src/data/qwik/tools.ts

```diff
@@ -17,12 +18,38 @@ export const tools: Tool<typeof toolTags[number]>[] = [
 		href: "https://qwik.builder.io/playground/",
 		tags: ["development"],
 	},
+	{
+		name: "Vite",
+		author: "Yuxi (Evan) You and Vite contributors",
+		description: "Next Generation Frontend Tooling",
+		image: "https://vitejs.dev/logo.svg",
+		href: "https://github.com/vitejs/vite",
+		tags: ["development"],
+	},
+	{
+		name: "Storybook",
+		author: "Storybook",
+		href: "https://storybook.js.org/tutorials/intro-to-storybook/react/en/get-started/",
```

Comment:
I guess this link is for React, Isn't it?

Reply:
Yeah, you're right. @markshenouda since there is "no official documentation", do you think we could link here our article about using storybook with Qwik: https://www.thisdot.co/blog/how-to-setup-storybook-in-a-qwik-project?

Reply:
I think we can do that, but the author and the name should also be updated.

Reply:
@AllStackDev1 isn't the name and the author referred to who "created" the tool?